README.rst

```diff
@@ -39,7 +37,7 @@ order to increase the loading speed of the pages.
 
 The dependency list is therefore:
 
-- `python`_ (2.5 minimum)
+- `python`_ (3.7 minimum)
```

Comment:
travis.yml seems to still be testing 3.6 ;-)

Reply:
Thanks. Will remove.